entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -374,31 +532,112 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
 
-### ForSqlServerHasIndex replaced with HasIndex 
+**Why**
+
+Currently it is not valid to map a derived type to a different table. This change avoids breaking in the future when it becomes a valid thing to do.
+
+**Mitigations**
+
+Remove any attempts to map derived types to other tables.
+
+## ForSqlServerHasIndex replaced with HasIndex 
 
 [Tracking Issue #12366](https://github.com/aspnet/EntityFrameworkCore/issues/12366)
 
-`ForSqlServerHasIndex().ForSqlServerInclude()` provided a way to configure columns used with `INCLUDE`, now the same can be accomplished with `HasIndex().ForSqlServerInclude()`
+This change was introduced in EF Core 3.0-preview 3.
 
-## Breaks that should only impact database providers
+**Old behavior**
 
-Changes that we expect to only impact database providers are documented under [provider changes](../../providers/provider-log.md).
+Prior to EF Core 3.0, `ForSqlServerHasIndex().ForSqlServerInclude()` provided a way to configure columns used with `INCLUDE`.
+
+**New behavior**
+
+Starting with EF Core 3.0, using `Include` on an index is now supported at the relational level. Use `HasIndex().ForSqlServerInclude()`.
+
+**Why**
+
+This change was made to consolidate the API for indexes with `Includes` into one place for all database providers.
+
+**Mitigations**
+
+Use the new API, as shown above.
+
+## EF Core no longer sends pragmas for SQLite FK enforcement
+
+[Tracking Issue #12151](https://github.com/aspnet/EntityFrameworkCore/issues/12151)
+
+This change was introduced in EF Core 3.0-preview 3.
+
+**Old behavior**
+
+Prior to EF Core 3.0, EF Core would send `PRAGMA foreign_keys = 1` when a connection to SQLite is opened.
 
-## Breaks to the Microsoft.Data.Sqlite ADO.NET database provider
+**New behavior**
 
-* Microsoft.EntityFrameworkCore.Sqlite now depends on SQLitePCLRaw.bundle_e_sqlite3 instead of SQLitePCLRaw.bundle_green. This makes the version of SQLite used on iOS consistent with other platforms.
-* Removed SqliteDbContextOptionsBuilder.SuppressForeignKeyEnforcement(). EF Core no longer sends `PRAGMA foreign_keys = 1` when a connection is opened. Foreign keys are enabled by default in SQLitePCLRaw.bundle_e_sqlite3. If you're not using that, you can enable foreign keys by specifying `Foreign Keys=True` in your connection string.
+Starting with EF Core 3.0, EF Core no longer sends `PRAGMA foreign_keys = 1` when a connection to SQLite is opened.
+
+**Why**
+
+This change was made because EF Core uses `SQLitePCLRaw.bundle_e_sqlite3` by default, which in turn means that FK enforcement is switched on by default and does not need to be explicitly enabled each time a connection is opened.
+
+**Mitigations**
+
+Foreign keys are enabled by default in SQLitePCLRaw.bundle_e_sqlite3, which is used by default for EF Core. For other cases, foreign keys can be enabled by specifying `Foreign Keys=True` in your connection string.
+
+## Microsoft.EntityFrameworkCore.Sqlite now depends on SQLitePCLRaw.bundle_e_sqlite3
```

Comment:
Are we conflating Microsoft.EntityFrameworkCore.Sqlite changes with Microsoft.Data.Sqlite? If so, is it on purpose?

Reply:
@bricelam commented on the previous PR that these were intended to be listed as EF breaks.

Reply:
Yeah, I saw it just now. Thanks.